src/foam/doc/DocBrowser.js

```diff
@@ -633,8 +633,8 @@ foam.CLASS({
     function init() {
       // TODO: There should be some helper support to make this easier
       var w = this.window.open('', '', 'width=700, heigh=1000');
-      var window = foam.core.Window.create({window: w});
-      var browser = this.DocBrowser.create({path: this.initialClass}, window.__subContext__);
+      var globalThis = foam.core.Window.create({globalThis: w});
```

Comment:
This line is a mistake. The property is called 'window'. revert these two lines.